.gitignore

```diff
@@ -1,3 +1,4 @@
 rootfs/bin/boot
+rootfs/usr/bin/*
```

Comment:
opps My bad will remove it. But even though there is a git ignore endpoint while doing git push its still considering binaries in that direcotry